### DISCUSSION TITLE
Is there a way to reset a panel stack?

### DISCUSSION THREAD

**Discussion by austinchen12** [palantir/blueprint#5988] [Q&A]
I have a submit button and I want to go back to the initial stack when its clicked. Is this currently possible?

**Comment by adidahiya** [comment]
@austinchen12 you can use controlled mode and supply the whole `stack` of panels. In this mode, you can simply reset by sending `stack={[initialPanel]}`.

See: [PanelStack2 docs](https://blueprintjs.com/docs/#core/components/panel-stack2).
